.github/workflows/release.yml

```diff
@@ -24,20 +23,28 @@ jobs:
       - name: Build and test
         run: ./gradlew build
 
-      - name: Tag name and copy jar
+      - name: Get version
+        id: version
+        run:  echo ::set-output name=version::v$(./gradlew -q getVersion).${{ github.run_number }}
+
+      - name: Push new tag
+        uses: anothrNick/github-tag-action@1.26.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          CUSTOM_TAG: ${{ steps.version.outputs.version }}
+
+      - name: Copy jar
         id: names
```

Comment:
Outdated/not needed?

Reply:
It's used to generate the tag and also set the name of the jar. This is needed

Reply:
Well it's a no-change and not really part of the pr anyway

Reply:
It's just a bit weird that the name is copy jar but id is names

Reply:
True. Maybe rename it to something else? I'm on my phone lol